### PR TITLE
add cli suggestions when arguments missing

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds suggestions for some commands in the cg cli

**How to prepare for testing**:
1. install cg on stage of the hasta: `bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh cli-suggest`
1. activate stage: `us`

**How to test cg analysis**:
1. run following command: `cg analysis`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for analysis

**How to test cg analysis config**:
1. run following command: `cg analysis config`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for analysis

**How to test cg analysis panel**:
1. run following command: `cg analysis panel`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for analysis

**How to test cg analysis start**:
1. run following command: `cg analysis start`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for analysis

**How to test cg store**:
1. run following command: `cg store`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are done analysing 

**How to test cg upload**:
1. run following command: `cg upload`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for uploading

**How to test cg upload delivery report**:
1. run following command: `cg upload delivery-report`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for creating a delivery report

**How to test cg upload delivery report to scout**:
1. run following command: `cg upload delivery-report-scout`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for creating a delivery report

**How to test cg upload coverage**:
1. run following command: `cg upload coverage`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for uploading

**How to test cg upload genotypes**:
1. run following command: `cg upload genotypes`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for uploading

**How to test cg upload scout**:
1. run following command: `cg upload scout`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for uploading

**How to test cg upload validate**:
1. run following command: `cg upload validate`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for uploading

**How to test cg deliver inbox**:
1. run following command: `cg deliver inbox`

**Expected outcome**:
- [x] `cg` should give you a list of cases that are ready for delivery

Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt  
Thanks for filling in who performed the code review and the test!

This is |minor| **version bump** because it adds functionality but the previous functionality should be preserved

### CODE REVIEW

File: cg/cli/deliver.py [hunk 1]
```diff
@@ -17,46 +16,48 @@
 @click.pass_context
 def deliver(context):
     """Deliver stuff."""
-    pass
+    context.obj['db'] = Store(context.obj['database'])
+    context.obj['api'] = DeliverAPI(
```
Comment (ingkebil) [inline cg/cli/deliver.py]: Can we name the key 'deliver_api'? It prevents me having to read code and then scroll up to see what API we're actually referring to.

Comment (patrikgrenfeldt) [reply cg/cli/deliver.py]: That change makes for too long lines, Isn't it quite safe to assume that an api in whatever.py is an whatever_api?

Comment (ingkebil) [reply cg/cli/deliver.py]: The (additional) problem comes from that we initialize context.obj with the config (cg.yaml), which holds keys that might be overwritten if we are not careful. Even though this is a red herring in this topic it's got nothing to do with the 'api' key.

And yes, it is quite safe. My point was that for me as a person I see 'api' in all these files and I always have to know either the filename, look at the class signature, or look at the constructor to find out to what api we are referring to. Which I find unclear.

Comment (patrikgrenfeldt) [reply cg/cli/deliver.py]: fixed

**Comment by patrikgrenfeldt** [comment]
cg analysis:
image

cg analysis config:
image

cg analysis panel:
image

cg analysis start:
image

cg store analysis:
image

cg upload:
image

cg upload delivery-report:
image

cg upload delivery-report-to-scout:
image

coverage:
image

cg upload genotypes:
image

cg upload scout:
image

cg upload validate:
image

cg deliver inbox:
image

👍

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**